src/centralserver/admin-ui/src/components/ui/ContextualAlerts.vue

```diff
@@ -70,8 +70,26 @@
               {{ notification.errorObject }}
             </div>
 
-            <!-- Show the error metadata if it exists -->
-            <div v-for="meta in errorMetadata(notification)" :key="meta">
+            <!-- Special case for pin code validation -->
+            <div v-if="errorCode(notification) === 'weak_pin'">
+              <div>
+                {{
+                  $t(`error_code.${errorMetadata(notification)[0]}`) +
+                  `: ${errorMetadata(notification)[1]}`
+                }}
+              </div>
+              <div>
+                {{
+                  $t(`error_code.${errorMetadata(notification)[2]}`) +
+                  `: ${errorMetadata(notification)[3]}`
+                }}
+              </div>
+            </div>
+            <div
+              v-else
+              v-for="meta in errorMetadata(notification)"
```

Comment:
Styling warning: according to linter the `v-for` should go before `v-else`

The file has other styling warning as well. It should be formatted with `prettier`. The default formatter hotkey for IntelliJ IDEA Prettier plugin is `Ctrl + Alt + Shift + P`

Reply:
Thanks for these out. This was fixed in https://github.com/nordic-institute/X-Road/pull/1101/commits/4872b6c6dcbc4b80f571e61fa70dfd59c0a67694